pkg/storage/minio/minio.go

```diff
@@ -557,6 +557,28 @@ func (c *Client) PlaceFiles(ctx context.Context, bucketFolders []string, prefix
 			output.PrintEvent(fmt.Sprintf("%s Downloading file %s", ui.IconFile, f.Name))
 			c.Log.Infof("Getting file %s", f)
 			objectName := f.Name
+
```

Comment:
with the propossed suggestion it is quite clear what we are doing, we don't need such detailed comment in this case

Reply:
Removed the comment, commit message will contain issue link.